Maria-Edgeworth-Letters-Project/me-tei
# Comments

***I'm including everyone on this PR for informational purposes, but I only need one approval to merge (which I will do myself)***

## What does this Pull Request do?

This PR removes comments from our encoded letters and template. The majority of these comments play no role in our current process. Some comments are useful in initial entry (so they have been retained in the template). The final encoded letters should not have any comments remaining as they don't serve a role for potential users and aren't meant to be public facing. This is something we should work on making a practice while encoding.

## How should this be tested?

- Do the changes alter the meaning of the encoding? Or are they inaccurately representing the digitized letter?
- Is it still easy to use the TEI template for entry with the remaining comments? Should any additional comments in the template be removed (see suggestion in Additional Notes)?
- Does the XML validate in Oxygen?

## Additional Notes

One spot where these changes are potentially problematic is within the `<msDesc>` section. In the `<head>` element in particular, removing comments can leave random commas and spaces. These are here because no placeName or date or persName might've been present. I'd propose that we have the RAs go through the letters and remove empty tags as well as the commas and spaces.

Here's an example with an empty placeName and trailing comma/space:
```
            <msDesc xml:lang="en">
               <msIdentifier>
                  <repository>Beinecke Rare Book and Manuscript Library</repository>
                  <collection>Maria Edgeworth Collection</collection>
                  <idno/>
               </msIdentifier>
               <head>Letter from <persName ref="./Personography.xml#EdgeM1">
                     <forename>Maria</forename>
                     <surname>Edgeworth</surname>
                  </persName> to <persName ref="./Personography.xml#MileX1">Mrs. Miles</persName>,
                        <placeName/>, <date when="1812-08-14">August 14, 1812</date>
               </head>
               <physDesc/>
            </msDesc>
```

I also question whether the following should be in the template written in as a comment or if these instructions should simply be stressed in the TEI workflow. Potentially this could be added to the existing information on spelling within `<editorialDecl>` rather than being present as a comment. I've left this in for now, but would appreciate discussion on this.

```
            
```

If there are other areas where these changes cause additional work, please comment before approving.
